processes/responsible_disclosure_template.md

```diff
@@ -1,5 +1,7 @@
 This project participates in the Responsible Disclosure Policy program for the Node.js Security Ecosystem.
 
+To report a Security Vulnerability [click here](https://hackerone.com/nodejs-ecosystem)
```

Comment:
How about this?:

```markdown
To report a security vulnerability, [use hackerone](https://hackerone.com/nodejs-ecosystem).
```

Reply:
I'm not a native English speaker but I was looking for something more inviting that calls for action. For me when I read "use hackerone" it gives me the feeling that I'm redirected elsewhere and it's someone else's responsibility.

Maybe we can try different phrases like one of those:


- [ ] Vulnerabilities can be reported directly to us via [HackerOne](https://hackerone.com/nodejs-ecosystem)
- [ ] 🚔Security Vulnerabilities can be reported directly to us via [HackerOne](https://hackerone.com/nodejs-ecosystem)
- [ ] Report a security vulnerabilities to us through the [HackerOne platform](https://hackerone.com/nodejs-ecosystem)

which in your opinion sounds better or maybe suggest a different wording?

Reply:
Any of your three suggestions work for me. Another idea: Make it a single bullet-point call-to-action:

```markdown
* [Report a security vulnerability](https://hackerone.com/nodejs-ecosystem)
```

...which would render something like this:

* [Report a security vulnerability](https://hackerone.com/nodejs-ecosystem)

Anyway, all of those work for me, so whatever you and/or others prefer!

Reply:
Thanks Rich, that suggestion sounds good! I'll go ahead and update for it.